docs/gemini/gemini.rst

```diff
@@ -0,0 +1,123 @@
+                     119.9986           Full Frame                         -- Gemini-North ...
+                     119.9983           Full Frame                         -- Gemini-North ...
+                      99.9983           Full Frame                         -- Gemini-North ...
+
+
+Observation Name Queries
+------------------------
+
+You may also do a query by the name of the object you are interested in.
+
+.. code-block:: python
+
+                >>> from astroquery.gemini import Observations
+
+                >>> data = Observations.query_object(objectname='m101')
+                >>> print(data[0:5])
+
+                exposure_time detector_roi_setting detector_welldepth_setting  telescope   ...
+                ------------- -------------------- -------------------------- ------------ ...
+                     119.9986           Full Frame                         -- Gemini-North ...
+                     119.9983           Full Frame                         -- Gemini-North ...
+                     119.9986           Full Frame                         -- Gemini-North ...
+                     119.9983           Full Frame                         -- Gemini-North ...
+                      99.9983           Full Frame                         -- Gemini-North ...
+
+
+Observation Criteria Queries
+----------------------------
+
+Additional search terms are available as optional arguments to the `~astroquery.gemini.ObservationsClass.query_criteria`
+call.  These all have default values of None, in which case they will not be considered during the search.
+
+The full API is in the api documentation, but some examples are the instrument used, such as GMOS-N, the
+observation_type, such as BIAS, and the program ID.
+
+.. code-block:: python
+                
+                >>> from astroquery.gemini import Observations
+
+                >>> data = Observations.query_criteria(instrument='GMOS-N',
+                ...                                    program_id='GN-CAL20191122',
+                ...                                    observation_type='BIAS')
+                >>> print(data[0:5])
+
+                exposure_time detector_roi_setting detector_welldepth_setting  telescope   mdready ...
+                ------------- -------------------- -------------------------- ------------ ------- ...
+                          0.0        Central Stamp                         -- Gemini-North    True ...
+                          0.0           Full Frame                         -- Gemini-North    True ...
+                          0.0           Full Frame                         -- Gemini-North    True ...
+                          0.0           Full Frame                         -- Gemini-North    True ...
+                          0.0           Full Frame                         -- Gemini-North    True ...
+
+
+Observation Raw Queries
+----------------------------
```

Comment:
please match the number of `-`'s to the length of the title string

Reply:
done - thanks!